src/gtk/dataview.cpp

```diff
@@ -3959,7 +3959,45 @@ bool wxDataViewCtrlInternal::ItemAdded( const wxDataViewItem &parent, const wxDa
 {
     if (!m_wx_model->IsVirtualListModel())
     {
-        wxGtkTreeModelNode *parent_node = FindNode( parent );
+        ItemList list;
```

Comment:
It would be nice to have a comment here explaining what this code does and why as it's definitely not immediately obvious.